platform/linux-generic/include/odp/api/plat/packet_inlines.h

```diff
@@ -443,6 +444,17 @@ _ODP_INLINE uint32_t odp_packet_buf_data_offset(odp_packet_buf_t pkt_buf)
 			(uintptr_t)odp_packet_buf_head(pkt_buf);
 }
 
+_ODP_INLINE void odp_packet_buf_data_set(odp_packet_buf_t pkt_buf, uint32_t data_offset,
+					 uint32_t data_len)
+{
+	uint8_t *head = (uint8_t *)odp_packet_buf_head(pkt_buf);
```

Comment:
Is this cast from void* to uint8_t* needed?

Reply:
It's required for C++ compatibility.